test/thetest.proto

```diff
@@ -386,6 +386,22 @@ message AnotherNinOptEnumDefault {
 	optional AnotherTestEnum Field1 = 1 [default=E];
 }
 
+// YetAnotherTestEnum is used to test cross-package import of custom name
```

Comment:
If you are going to declare these, you should probably also use them in a message, that way you can get some free combo testing :)